includes/rest/class-inbox.php

```diff
@@ -241,6 +233,16 @@ public static function user_inbox_post_parameters() {
 			'sanitize_callback' => 'esc_url_raw',
 		);
 
+		$params['signature'] = array(
```

Comment:
this will check if the JSON blob will have an attribute `signature` and should therefore always fail!?!

I would assume that this is the wrong place to verify HTTP headers.

Reply:
I have updated the code locally, will push today